File: program/program/print-string-of-ascii-values/PrintStringOfAsciiValues.rb
def ascii_to_string(ascii_values)
  characters = ascii_values.split.map { |ascii| ascii.to_i.chr }
  return characters.join
end

# Input ASCII values separated by spaces
print "Enter ASCII values separated by spaces: "
ascii_values = gets.chomp

begin
  result = ascii_to_string(ascii_values)
  puts "Output: #{result}"
rescue ArgumentError => e
  puts "Error: #{e.message}"
end

File: program/program/print-string-of-ascii-values/print_string_of_ascii_values.r
# Function to convert ASCII values to characters
ascii_to_char <- function(ascii_values) {
  characters <- sapply(ascii_values, function(x) rawToChar(as.raw(x)))
  return(paste(characters, collapse = ""))
}

# Input ASCII values as a string separated by space
input <- readline("Enter ASCII values separated by space: ")

# Convert input string to a numeric vector of ASCII values
ascii_values <- as.numeric(strsplit(input, " ")[[1]])

# Call the ascii_to_char function
output <- ascii_to_char(ascii_values)

# Print the output
cat("Output:", output, "\n")
